plugins/vuetify.js

```diff
@@ -3,6 +3,7 @@ import Vuetify from 'vuetify/lib'
 import colors from 'vuetify/es5/util/colors'
 
 Vue.use(Vuetify, {
+  defaultAssets: false,
```

Comment:
Prevents Vuetify from loading default font & icons since we have already previously loaded these.
Essentially, don't load extra resources that we already have.